eclass/distutils-r1.eclass

```diff
@@ -1203,6 +1212,13 @@ distutils-r1_python_compile() {
 		else
 			esetup.py build -j "${jobs}" "${@}"
 		fi
+	elif [[ ${DISTUTILS_USE_PEP517} == maturin ]]; then
```

Comment:
How about turning this into `case`?

Reply:
Sure, will need to re-indent the above block & will conflict with pending build_ext patch though.

Reply:
Please rebase on top of it, and close my PR (so it'll get merged along with yours).

Reply:
Alright, should be sorted out. Don't hesitate to say if any small things left to adjust.